<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabSpecies</class>
 <widget class="QWidget" name="TabSpecies">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Species</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="0">
        <widget class="QPushButton" name="btnAddSpecies">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btnRemoveSpecies">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTreeWidget" name="listSpecies">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="lblSpecies">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Species:</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="grpSpecies">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Species Settings</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridSpeciesSettings">
         <item row="0" column="0">
          <widget class="QLabel" name="lblSpeciesName">
           <property name="text">
            <string>Name:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="lblSpeciesColourLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>The colour that is used to display the concentration of this species</string>
           </property>
           <property name="statusTip">
            <string>The colour that is used to display the concentration of this species</string>
           </property>
           <property name="whatsThis">
            <string>The colour that is used to display the concentration of this species</string>
           </property>
           <property name="text">
            <string>Colour:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblSpeciesIsSpacial">
           <property name="toolTip">
            <string>'Spatial': species concentration can vary depending on the location within the compartment</string>
           </property>
           <property name="statusTip">
            <string>'Spatial': species concentration can vary depending on the location within the compartment</string>
           </property>
           <property name="whatsThis">
            <string>&lt;h4&gt;Spatial&lt;/h4&gt;&lt;p&gt;A spatial species has a concentration that is a function of space within the compartment, i.e. it can take different values at different locations within the compartment&lt;/p&gt;</string>
           </property>
           <property name="text">
            <string>Spatial:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="6">
          <widget class="QLabel" name="lblDiffusionConstantUnits">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="6">
          <widget class="QCheckBox" name="chkSpeciesIsConstant">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>if &quot;Constant&quot; is checked: the species has a fixed concentration during the simulation</string>
           </property>
           <property name="statusTip">
            <string>'Constant': species concentration cannot be altered by the simulation</string>
           </property>
           <property name="whatsThis">
            <string>&lt;h4&gt;Constant&lt;/h4&gt;&lt;p&gt;A constant species has a fixed concentration. It keeps its initial concentration for the whole simulation.&lt;/p&gt;</string>
           </property>
           <property name="text">
            <string>species concentration is fixed throughout the whole simulation</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QRadioButton" name="radInitialConcentrationAnalytic">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Spatially varying initial concentration: varies with position</string>
           </property>
           <property name="statusTip">
            <string>Spatially varying initial concentration: varies with position</string>
           </property>
           <property name="whatsThis">
            <string>&lt;h4&gt;Spatially varying initial concentration&lt;/h4&gt;
&lt;p&gt;The initial concentration of the species can vary depending on the location within the compartment.&lt;/p&gt;</string>
           </property>
           <property name="text">
            <string>Analytic expression</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblSpeciesCompartment">
           <property name="text">
            <string>Compartment:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="7">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="7" column="2" colspan="5">
          <widget class="QPushButton" name="btnEditImageConcentration">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Import an image of the spatial concentration of this species</string>
           </property>
           <property name="statusTip">
            <string>Import an image of the spatial concentration of this species</string>
           </property>
           <property name="whatsThis">
            <string>&lt;h4&gt;import from image&lt;/h4&gt;&lt;p&gt;Import an image of the spatial concentration of this species&lt;/p&gt;</string>
           </property>
           <property name="text">
            <string>edit image...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblSpeciesIsConstant">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>A constant species has a fixed concentration</string>
           </property>
           <property name="statusTip">
            <string>A constant species has a fixed concentration</string>
           </property>
           <property name="whatsThis">
            <string>&lt;h4&gt;Constant&lt;/h4&gt;&lt;p&gt;A constant species has a fixed concentration. It keeps its initial concentration for the whole simulation.&lt;/p&gt;</string>
           </property>
           <property name="text">
            <string>Constant:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="lblDiffConst">
           <property name="toolTip">
            <string>Diffusion constant: how fast the species diffuses (spreads out) within the compartment</string>
           </property>
           <property name="statusTip">
            <string>Diffusion constant: how fast the species diffuses (spreads out) within the compartment</string>
           </property>
           <property name="whatsThis">
            <string>&lt;h4&gt;Diffusion constant&lt;/h4&gt;
&lt;p&gt;The rate at which the species diffuses within the compartment&lt;/p&gt;</string>
           </property>
           <property name="text">
            <string>Diffusion Constant:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="6">
          <widget class="QCheckBox" name="chkSpeciesIsSpatial">
           <property name="toolTip">
            <string>'Spatial': species concentration can vary depending on the location within the compartment</string>
           </property>
           <property name="statusTip">
            <string>'Spatial': species concentration can vary depending on the location within the compartment</string>
           </property>
           <property name="whatsThis">
            <string>&lt;h4&gt;Spatial&lt;/h4&gt;&lt;p&gt;A spatial species has a concentration that is a function of space within the compartment, i.e. it can take different values at different locations within the compartment&lt;/p&gt;</string>
           </property>
           <property name="text">
            <string>species concentration has a spatial dependence</string>
           </property>
          </widget>
         </item>
         <item row="5" column="6">
          <widget class="QLabel" name="lblInitialConcentrationUnits">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="2" colspan="4">
          <widget class="QLineEdit" name="txtInitialConcentration">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="6">
          <widget class="QLineEdit" name="txtSpeciesName"/>
         </item>
         <item row="10" column="6">
          <widget class="QPushButton" name="btnChangeSpeciesColour">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Change the colour that is used to display the concentration of this species</string>
           </property>
           <property name="statusTip">
            <string>Change the colour that is used to display the concentration of this species</string>
           </property>
           <property name="whatsThis">
            <string>Change the colour that is used to display the concentration of this species</string>
           </property>
           <property name="text">
            <string>change colour...</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QRadioButton" name="radInitialConcentrationUniform">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Spatially uniform initial concentration: no spatial dependence</string>
           </property>
           <property name="statusTip">
            <string>Spatially uniform initial concentration: no spatial dependence</string>
           </property>
           <property name="whatsThis">
            <string>&lt;h4&gt;Spatially uniform initial concentration&lt;/h4&gt;
&lt;p&gt;The initial concentration of the species is the same everywhere in the compartment.&lt;/p&gt;</string>
           </property>
           <property name="text">
            <string>Spatially uniform</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="10" column="1" colspan="5">
          <widget class="QLabel" name="lblSpeciesColour">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>The colour that is used to display the concentration of this species</string>
           </property>
           <property name="statusTip">
            <string>The colour that is used to display the concentration of this species</string>
           </property>
           <property name="whatsThis">
            <string>The colour that is used to display the concentration of this species</string>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>none</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="6">
          <widget class="QComboBox" name="cmbSpeciesCompartment"/>
         </item>
         <item row="6" column="2" colspan="5">
          <widget class="QPushButton" name="btnEditAnalyticConcentration">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>edit expression...</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QRadioButton" name="radInitialConcentrationImage">
           <property name="text">
            <string>Concentration image</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1" colspan="5">
          <widget class="QLineEdit" name="txtDiffusionConstant">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Diffusion constant: how fast the species diffuses (spreads out) within the compartment</string>
           </property>
           <property name="statusTip">
            <string>Diffusion constant: how fast the species diffuses (spreads out) within the compartment</string>
           </property>
           <property name="whatsThis">
            <string>&lt;h4&gt;Diffusion constant&lt;/h4&gt;
&lt;p&gt;The rate at which the species diffuses within the compartment&lt;/p&gt;</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="7">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lblInitConc">
           <property name="text">
            <string>Initial Concentration:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
